README.md

```diff
@@ -34,42 +34,50 @@ This fork aims to be community managed, meaning that the decisions are taken
 in a collegial fashion, and that a bunch of maintainers should be able to make
 the review process quicker and more streamlined.
 
-## Install
+## Installation and usage
 
-To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
-The oldest Node.js version we support is 4.2.0.
+The Lounge requires [Node.js](https://nodejs.org/) v4 or more recent.
 
-If you still use 0.10 or 0.12 we strongly advise you to upgrade before installing The Lounge.
-For more information on how to upgrade, read the [documentation](https://nodejs.org/en/download/package-manager/).
+### Running stable releases from npm (recommended)
 
-```
+Run this in a terminal to install (or upgrade) the latest stable release from
+[npm](https://www.npmjs.com/):
+
+```sh
 sudo npm install -g thelounge
 ```
 
-## Usage
-
-When the install is complete, go ahead and run this in your terminal:
+When install is complete, run:
 
+```sh
+lounge start
 ```
+
+For more information, read the [documentation](https://thelounge.github.io/docs/), [wiki](https://github.com/thelounge/lounge/wiki), or run:
+
+```sh
 lounge --help
 ```
 
-For more information, read the [documentation](https://thelounge.github.io/docs/) or [wiki](https://github.com/thelounge/lounge/wiki).
+### Running from source
 
-## Development setup
-
-To run the app from source, just clone the code and run this in your terminal:
+The following commands install the development version of The Lounge. A word of
+caution: while it is the most recent codebase, this is not production-ready!
 
-```
+```sh
+git clone https://github.com/thelounge/lounge.git
+cd lounge
```

Comment:
If command run is `git clone https://github.com/thelounge/lounge.git`, then the new directory is going to be `lounge` in the current one, so running these 2 commands altogether should always be good.
If people decide to run `git clone https://github.com/thelounge/lounge.git my-custom-dir`, I believe them to be advanced enough to adapt the `cd` to their needs 😅.

Idea is by default you can just copy and paste these in a terminal and it should work as is.